content/blog/urbit-is-for-communities.md

```diff
@@ -0,0 +1,97 @@
++++
+title = "Urbit is for Communities"
+date = 2020-03-23
+description = "Urbit is for giving communities the tools to evolve their own environments; for us all to feel a sense of life and self-directedness in the digital world."
```

Comment:
The exact wording I liked was "shape their own environments" — evolve seems not aggressive enough. idk, I'm being pedantic, clearly. @galenwp

Reply:
I'm fine with switching to 'shape'. Although if you read the piece, the argument has to do not just with shaping, but shaping over time.